src/test/java/uk/gov/companieshouse/api/testdata/service/impl/AcspProfileServiceImplTest.java

```diff
@@ -45,20 +52,47 @@ class AcspProfileServiceImplTest {
     @Captor
     private ArgumentCaptor<AcspProfile> profileCaptor;
 
-    private AcspProfile createSavedProfile() {
-        AcspProfile profile = new AcspProfile();
-        profile.setAcspNumber("randomId");
-        return profile;
+    private AcspProfile savedProfile;
```

Comment:
You might consider following the Java naming convention by using a lowercase version of the class name for the instance variable. This can make the code more readable for others, but it's entirely optional. Again the code works as is its so personal choice.
`private AcspProfile acspProfile;`
`private AcspProfileSpec acspProfileSpec;`